src/discord/commands/create_calendar.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{calendar::Client as CalendarClient, Error};
+use serenity::all::{Context, CreateCommand, GuildId, Permissions, ResolvedOption};
+use tracing::{info, instrument};
+
+use crate::calendar::get_calendar_url;
+
+use super::MessageResult;
+
+#[instrument]
+pub async fn run(
+    ctx: &Context,
+    guild_id: GuildId,
+    _options: &[ResolvedOption<'_>],
+) -> MessageResult {
+    info!("Creating a calendar");
+    let lock = ctx.data.read().await;
+    let calendar_client = lock
+        .get::<CalendarClient>()
+        .ok_or(Error::NoCalendarClient)?;
+    let calendars = calendar_client.get_calendars_by_guild_id(&guild_id).await?;
+    if calendars.is_some() {
+        return Ok(format!(
+            "A calendar already exists: {}",
+            calendars
+                .into_iter()
+                .map(|calendar| {
+                    let calendar_id = calendar.id.expect("No calendar id");
+                    get_calendar_url(&calendar_id)
+                })
+                .collect::<Vec<_>>()
+                .join(", ")
+        ));
+    }
+    calendar_client
+        .create_calendar(&guild_id.to_string())
+        .await?;
```

Comment:
Would be cool to name the calendar in a human-readable format rather than the guild_id, if that's possible